src/SonarQube.Bootstrapper/app.config

```diff
@@ -4,7 +4,7 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral"/>
-        <bindingRedirect oldVersion="0.0.0.0-6.0.0.0" newVersion="6.0.0.0"/>
+        <bindingRedirect oldVersion="0.0.0.0-8.0.0.0" newVersion="8.0.0.0"/>
```

Comment:
Most projects used version 8, some used 6...

Reply:
Might be worth updating to last version (maybe in a latter commit).

Reply:
It's on my list. Didn't want to do it at this time, as there are already quite a few changes.